### PR TITLE
chore(views): removed duplicate class name for plugin object

### PR DESCRIPTION
**Pull request by jdalsem**

### CODE REVIEW

File: views/default/object/plugin/full.php [hunk 1]
```diff
@@ -111,7 +111,7 @@
 	$options['class'] = 'elgg-button elgg-button-cancel';
 	$options['text'] = elgg_echo('admin:plugins:deactivate');
 	if (!$can_activate) {
```
Comment (mrclay) [inline views/default/object/plugin/full.php]: How is this case even possible? Active but can't be activated.

Comment (jdalsem) [inline views/default/object/plugin/full.php]: a) active first with all dependencies met... remove a plugin
b) manual deactivation
c) some other reason... not very hard to come by though

Comment (jdalsem) [inline views/default/object/plugin/full.php]: d) reorder (it does not validate dependencies)

**Comment by mrclay** [comment]
If the plugin really is still active I think leaving the elgg-state-active class in place makes sense.

**Comment by mrclay** [comment]
(if there's a style problem we should rather fix the CSS)

**Comment by jdalsem** [comment]
this chore only fixes elgg-state-active being added twice in this loop... no change in logic

**Comment by mrclay** [comment]
I see. Thanks.

**Pull request merged**